apps/retrospective/services/sprint_sync.go

```diff
@@ -63,9 +65,29 @@ func (service SprintService) SyncSprintData(sprintID string) (err error) {
 	var timeTrackerTaskKeys []string
 	var timeLogs []timeTrackerSerializers.TimeLog
 	sprintMemberTimeLogs := map[uint][]timeTrackerSerializers.TimeLog{}
+	var taskProviderConfigList []interface{}
+
+	json.Unmarshal(taskProviderConfig, &taskProviderConfigList)
+	tpConfig := taskProviderConfigList[0]
+	tp := tpConfig.(map[string]interface{})
+
 	for _, sprintMember := range sprint.SprintMembers {
 		var memberTaskKeys []string
 		memberTaskKeys, timeLogs, err = service.GetSprintMemberTimeTrackerData(sprintMember, sprint)
+
+		if tp["type"].(string) == constants.PivotalTracker {
```

Comment:
Move this logic to Pivotal tracker providers code.

Reply:
done